README.md

```diff
@@ -1,4 +1,681 @@
+array of JSON objects, where each object represents a Kafka topic and has the
+following properties:
+
+| Property name     | Description                                             |
+|-------------------|---------------------------------------------------------|
+| name              | The name of the Kafka topic.                            |
+| partitions        | The number of partitions assigned to the Kafka topic.   |
+| retentionsMs      | The retention period for messages on the topic (in ms). |
+| cleanupPolicy     | The cleanup policy of the Kafka topic.                  |
+
+#### Example
+
+```golang
+	// Construct an instance of the ListTopicsOptions model
+	listTopicsOptionsModel := new(ibmeventstreamsadminrestapiv1.ListTopicsOptions)
+
+	// Call ListTopics.
+	result, response, operationErr := serviceAPI.ListTopics(listTopicsOptionsModel)
+	if operationErr != nil {
+		return fmt.Errorf("Error Listing Topics" + operationErr.Error())
+	}
+
+	// Check the result.
+	if response.StatusCode != http.StatusOK {
+		return fmt.Errorf("Error Listing Topics: status %d\n", response.StatusCode)
+	}
+
+	// Loop and print topics.
+	for _, topicDetail := range result {
+		fmt.Printf("\tname: %s\n", *topicDetail.Name)
+	}
+```
+
+### Getting a Kafka topic
+
+To get a Kafka topic detail information, issue a GET request to the `/admin/topics/TOPICNAME`
+path (where `TOPICNAME` is the name of the Kafka topic that you want to get).  
+
+Expected status codes
+  - 200: Retrieve topic details successfully in following format:
+```json
+{
+  "name": "MYTOPIC",
+  "partitions": 1,
+  "replicationFactor": 3,
+  "retentionMs": 86400000,
+  "cleanupPolicy": "delete",
+  "configs": {
+    "cleanup.policy": "delete",
+    "min.insync.replicas": "2",
+    "retention.bytes": "1073741824",
+    "retention.ms": "86400000",
+    "segment.bytes": "536870912"
+  },
+  "replicaAssignments": [
+    {
+      "id": 0,
+      "brokers": {
+        "replicas": [
+          3,
+          2,
+          4
+        ]
+      }
+    }
+  ]
+}
+```
+
+#### Example
+```golang
+	// Construct an instance of the GetTopicOptions model
+	getTopicOptionsModel := new(ibmeventstreamsadminrestapiv1.GetTopicOptions)
+	getTopicOptionsModel.TopicName = core.StringPtr("test-topic")
+	getTopicOptionsModel.Headers = map[string]string{"x-custom-header": "x-custom-value"}
+
+	// Call List Topic Details.
+	result, response, operationErr := serviceAPI.GetTopic(getTopicOptionsModel)
+	if operationErr != nil {
+		return fmt.Errorf("Error Listing Topic Details" + operationErr.Error())
+	}
+
+	// Check the result.
+	if response.StatusCode != http.StatusOK {
+		return fmt.Errorf("Error Listing Topic Details: status %d\n", response.StatusCode)
+	}
+
+	// Print topics details.
+	fmt.Printf("\tname: \t\t\t%s\n", *result.Name)
+
+	// The number of partitions.
+	fmt.Printf("\tno of partitions: \t%d\n", *result.Partitions)
+
+	// The number of replication factor.
```

Comment:
`The number of replication factor.` -> The replication factor.`. I'm actually not sure these comments have a lot of value.

Reply:
agreed, removed from example